q2_phylogeny/_fasttree.py

```diff
@@ -21,14 +22,22 @@ def run_command(cmd, output_fp, verbose=True):
               "no longer exist.")
         print("\nCommand:", end=' ')
         print(" ".join(cmd), end='\n\n')
+
+    if env is None:
+        env = os.environ.copy()
```

Comment:
This env copy isn't necessary, you can just pass `env` through to `subprocess.run` if it's `None`. From `Popen` docs:

> If env is not None, it must be a mapping that defines the environment variables for the new process; these are used instead of the default behavior of inheriting the current process’ environment.